src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_app_permission_consent_notification.cc

```diff
@@ -50,6 +50,7 @@ namespace {
 struct PermissionsAppender
     : public std::unary_function<void,
                                  const smart_objects::SmartArray::value_type&> {
+  // cppcheck-suppress noExplicitConstructor
   PermissionsAppender(policy::PermissionConsent& consents)
```

Comment:
Mark all constructors with one argument as explicit

Reply:
FIxed in caee951663139c03dfb7132f1ba82121580de410
Please check.